enhancements/api-review/gcp_user_defined_labels.md

```diff
@@ -0,0 +1,301 @@
+---
+title: apply-user-defined-labels-to-all-gcp-resources-created-by-openShift
+authors:
+  - "@bhb"
+reviewers:
+  - @patrickdillon
```

Comment:
The linter is failing because of the unquoted `@` here. The names need to be in double quotes.

Reply:
Thank you @dhellmann !!